.travis.yml

```diff
@@ -1,13 +1,18 @@
 sudo: required
 language: node_js
 node_js:
-  - "0.10.25"
+  - "6"
```

Comment:
@johncrossman - exactly, latest stable 6.* https://docs.travis-ci.com/user/languages/javascript-with-nodejs

We could nail it down more specifically, but since this is a test-only environment it might be good to have advance warning of breaking changes.